common/test/acceptance/tests/studio/test_studio_help.py

```diff
@@ -3,10 +3,26 @@
 """
 
 from flaky import flaky
+from bok_choy.web_app_test import WebAppTest
 
 from common.test.acceptance.tests.studio.base_studio_test import StudioCourseTest
 from common.test.acceptance.pages.studio.index import DashboardPage
-from common.test.acceptance.pages.studio.utils import click_studio_help, studio_help_links
+from common.test.acceptance.pages.studio.utils import click_studio_help, studio_help_links, get_element
+from common.test.acceptance.pages.studio.index import IndexPage, HomePage, DashboardPageWithPrograms
+from common.test.acceptance.tests.studio.base_studio_test import StudioLibraryTest
+from common.test.acceptance.pages.studio.utils import click_css
+from common.test.acceptance.pages.studio.library import LibraryPage
+from common.test.acceptance.pages.studio.users import LibraryUsersPage
+from common.test.acceptance.tests.helpers import (
+    assert_links,
+    assert_help_is_open
+)
+from common.test.acceptance.pages.studio.import_export import ExportLibraryPage, ImportLibraryPage
+from common.test.acceptance.pages.studio.auto_auth import AutoAuthPage
+
+
+NAV_HELP_CSS = '.nav-item.nav-account-help a'
+SIDE_BAR_HELP_CSS = '.bit.external-help a'
```

Comment:
@Muddasser See my comment regarding moving the css selectors to page object file.